src/renderer/components/+cluster-settings/components/cluster-accessible-namespaces.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { observer } from "mobx-react";
+import { Cluster } from "../../../../main/cluster";
+import { SubTitle } from "../../layout/sub-title";
+import { EditableList } from "../../editable-list";
+import { observable } from "mobx";
+import { _i18n } from "../../../i18n";
+
+interface Props {
+  cluster: Cluster;
+}
+
+@observer
+export class ClusterAccessibleNamespaces extends React.Component<Props> {
+  @observable namespaces = new Set(this.props.cluster.accessibleNamespaces);
+
+  render() {
+    return (
+      <>
+        <SubTitle title="Cluster's Accessible Namespaces" />
```

Comment:
Maybe just leave with `Accessible Namespaces` to keep it simple since we are looking Cluster Settings?